system/jlib/jtrace.hpp

```diff
@@ -55,6 +66,33 @@ class jlib_decl LogTrace
     void setLocalId(const char* id);
 };
 
+//class CHPCCHttpTextMapCarrier;
+
+interface ISpan : extends IInterface
+{
+    virtual void setSpanAttribute(const char * key, const char * val) = 0;
+    virtual void setSpanAttributes(const IProperties * attributes) = 0;
+    virtual void addSpanEvent(const char * eventName) = 0;
+
+    virtual void querySpanContextProperties(IProperties * contextProps) = 0;
```

Comment:
consistentency: Functions that start query generally return a value.  get is generally used if returns a value in a supplied parameter.